### DISCUSSION TITLE
v2.3.0

### DISCUSSION THREAD

**Discussion by mochi-co** [mochi-mqtt/server#272] [Announcements]
## We have moved!
Mochi MQTT has become an organisation! 🥳
**https://github.com/mochi-co/mqtt has moved to our new organisation: https://github.com/mochi-mqtt/server**

See #271 for more information.

## What's Changed
* Migrate imports, copyrights, etc by @mochi-co in https://github.com/mochi-mqtt/server/pull/270
  * All imports have been updated to the new organisation repo address. 
  * Copyright and licensing has been updated to cover mochi-mqtt organisation.
  * Minor changes in readme paths and references.

**Full Changelog**: https://github.com/mochi-mqtt/server/compare/v2.2.16...v2.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/mochi-mqtt/server/releases/tag/v2.3.0'>v2.3.0</a>.</em>

**Comment by prosenjitjoy** [comment]
last good version to support zerolog
